### PR TITLE
Update amp-video doc for click-to-play pattern

### PR DESCRIPTION
**Pull request by aghassemi**
Closes https://github.com/ampproject/amphtml/issues/8896

### CODE REVIEW

File: extensions/amp-video/amp-video.md [hunk 1]
```diff
@@ -103,6 +101,12 @@ If present, will automatically loop the video back to the start upon reaching th
 
 This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
 
+## Click-to-Play overlay
+Click-to-play is a common UX for video players on the web. `amp-video` supports
```
Comment (bpaduch) [inline extensions/amp-video/amp-video.md]: add "feature" after UX

File: extensions/amp-video/amp-video.md [hunk 2]
```diff
@@ -103,6 +101,12 @@ If present, will automatically loop the video back to the start upon reaching th
...
+`play` standard AMP action which allows us to implement this pattern overlays with ease.
```
Comment (bpaduch) [inline extensions/amp-video/amp-video.md]: Rewrite as:

the standard `play` AMP action, allowing you to implement click-to-play easily.

File: extensions/amp-video/amp-video.md (hunk 1 above)
Comment (aghassemi) [inline extensions/amp-video/amp-video.md]: done

File: extensions/amp-video/amp-video.md (hunk 2 above)
Comment (aghassemi) [inline extensions/amp-video/amp-video.md]: done

**Pull request merged**